src/solvers/dgsem_tree/dg_2d_subcell_limiters.jl

```diff
@@ -47,18 +69,20 @@ end
 
 @inline function subcell_limiting_kernel!(du, u,
                                           element, mesh::TreeMesh{2},
-                                          nonconservative_terms::False, equations,
+                                          nonconservative_terms, equations,
                                           volume_integral, limiter::SubcellLimiterIDP,
                                           dg::DGSEM, cache)
```

Comment:
If we don't dispatch anymore on `nonconservative_terms`, we don't need it anymore as argument of this function, do we?

Reply:
We still need it since it is an argument of three functions that are called in this routine: `calcflux_fhat!`, `calcflux_fv!`, and `calcflux_antidiffusive!`.